sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -421,7 +421,7 @@ Page<string> NextPageFunc(string nextLink, int? pageSizeHint)
         }
 
         /// <summary>
-        /// Gets all the relationships referencing a digital twin as a target.
+        /// Gets all the relationships referencing a digital twin as a target asynchronously. After the task completes response would be set to <see cref="AsyncPageable{T}"/>.
```

Comment:
This shouldn't really say `After the task completes ...`. Iterating through the `AsyncPageable` is an async operation. We should probably highlight that as well.

Reply:
Good catch. Will update all the comments for the pageable responses.